po/R-fr.po

```diff
@@ -0,0 +1,645 @@
+msgstr ""
+"Le nombre de colonnes ciblées ne correspond pas au nombre de formats qui "
+"sont forunis"
+
+#: date_standardization_helpers.R:572
+msgid ""
+"Only one format is needed if all target columns contain values of the same "
+"format. Otherwise, one format per target column must be provided."
+msgstr ""
+"Seul un format est requis si les colonnes ciblées ont des valeures du même "
+"format. Sinon, vous devez fornir un format par colonne cible."
+
+#: date_standardization_helpers.R:577
+msgid "Target columns will be standardized using the format: {.val {format}}."
+msgstr ""
+"Les colonnes ciblées seront standardisées en utilisant le format suivant: {."
+"val {format}}."
+
+#: default_cleanepi_settings.R:48
+msgid ""
+"Found the following unrecognised arguments to {.fn clean_data}: {.emph "
+"{toString(extra)}}."
+msgstr ""
+"{.fn clean_data} ne reconnait pas les arguments suivants: {.emph "
+"{toString(extra)}}."
+
+#: default_cleanepi_settings.R:49
+msgid "{.fn clean_data} does not support arguments other than the defaults."
+msgstr ""
+"{.fn clean_data} ne prend pas d'arguments autre que ceux qui sont définis "
+"par défault."
+
+#: default_cleanepi_settings.R:50
+msgid "You provided unexpected arguments."
+msgstr "Vous avez fourni des arguments qui ne sont pas prévus."
+
+#: default_cleanepi_settings.R:51
+msgid ""
+"Run {.code get_default_params()} to display the list of default parameters."
+msgstr ""
+"Entrez {.code get_default_params()} pour consulter la list des arguments "
+"prédéfinis."
+
+#: dictionary_based_cleaning.R:48
+msgid "Incorrect data dictionary."
+msgstr "Dictionnaire de données incorrect."
+
+#: dictionary_based_cleaning.R:49
+msgid ""
+"The value for the {.emph dictionary} argument must be a {.cls data.frame} "
+"with the following columns: {.field {toString(all_columns)}}."
+msgstr ""
+"La valeur fournie a l'argument {.emph dictionary} doit être un {.cls data."
+"frame} contenant les colonnes suivantes: {.field {toString(all_columns)}}."
+
+#: dictionary_based_cleaning.R:50
+msgid ""
+"The following columns are mandatory: {.field {toString(mandatory_columns)}}."
+msgstr ""
+"Les colonnes suivantes sont obligatoires: {.field "
+"{toString(mandatory_columns)}}."
+
+#: dictionary_based_cleaning.R:58
+msgid "Incorrect column names {.field grp} in the data dictionary."
+msgstr ""
+"Des noms de colonnes incorrects ont été trouvés dans la colonne {.field grp} "
+"du dictionnaire de données."
+
+#: dictionary_based_cleaning.R:59
+msgid ""
+"Values in {.field grp} column of the data dictionary must be found in the "
+"input data frame."
+msgstr ""
+"Les noms de colonnes fournies dans la colonne {.field grp} du dictionnaire "
+"de données doivent figurer dans les données d'entrée."
+
+#: dictionary_based_cleaning.R:60
+msgid "Did you enter an incorrect column name?"
+msgstr "Avez-vous fourni un nom de colonne incorrect?"
+
+#: dictionary_based_cleaning.R:71
+msgid "You can either:"
+msgstr "Vous pouvez soit:"
```

Comment:
```suggestion
msgstr "Vous pouvez soit :"
```

`:` takes a space before and after in French.

Please do a search in the full document here as well to correct all occurrences.

Reply:
I have accounted for this.